vllm/lora/models.py

```diff
@@ -384,8 +392,15 @@ def activate_adapter(
             module_lora = lora_model.get_lora(module_name)
             if module_lora:
                 module_lora.optimize()
+                # Bias is not explicitly enabled with the flag enable_lora_bias.
+                if not self.lora_config.bias_enabled:
+                    module_lora.bias = None
+                    raise ValueError(
+                        f"Adapter bias cannot be used for {module_name}"
+                        " without --enable-lora-bias.")
```

Comment:
This doesn't look right and is causing blanket lora failures. I think it should be:
```suggestion
                if module_lora.bias is not None and not self.lora_config.bias_enabled:
                    raise ValueError(
                        f"Adapter bias cannot be used for {module_name}"
                        " without --enable-lora-bias.")
```

Reply:
Incorporated the comment.